resources/css/bem/daily-challenge.less

```diff
@@ -5,9 +5,29 @@
   background: hsl(var(--hsl-b4));
   border-radius: @border-radius-large;
   min-width: 0;
-  display: flex;
-  align-items: center;
-  padding: 3px;
+  position: relative;
+
+  &--played-today {
+    border: 2px solid @osu-colour-lime-1;
```

Comment:
This is 1px on the [design file](https://www.figma.com/design/vpLHaQ7CcjUuPH2hZiZ3WE/Quick-Fix%2FDaily-Challenge-Streak) but I find that 1px looks terrible on the rounded corners:

| 1px | 2px |
| :-: | :-: |
| ![1736414918](https://github.com/user-attachments/assets/b6a95efd-fe82-4699-9617-bc801d0fce87) | ![1736414351](https://github.com/user-attachments/assets/f5c287f9-22f3-48ae-967c-bd2875b98dde) |

Arguably even 2px looks not great. Maybe 3px even could be considered. cc @ppy/team-design

Reply:
the extra spacing between the day count box and the green border seems a bit weird?

also dunno if matters but the popup box covers the check mark a little bit

Reply:
> the extra spacing between the day count box and the green border seems a bit weird?

I mean yeah maybe. I'm just following design direction here :shrug: 

> also dunno if matters but the popup box covers the check mark a little bit

Yeah it does and was also unsure if it mattered. I guess maybe the design peoples can say?

Just for visual reference here's a screenshot:

![1736416429](https://github.com/user-attachments/assets/45f9264a-16fb-47da-872f-f9b6a12a84bd)